WordPress/Classes/Utility/ImmuTable.swift

```diff
@@ -279,6 +279,14 @@ public class ImmuTableViewHandler: NSObject, UITableViewDataSource, UITableViewD
         return cell
     }
 
+    public func tableView(tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return viewModel.sections[section].headerText
+    }
+
+    public func tableView(tableView: UITableView, titleForFooterInSection section: Int) -> String? {
+        return viewModel.sections[section].footerText
```

Comment:
Sure! `WPTableViewSectionHeaderFooterView` was implemented to match Calypso's Headers / Footers. 
This helper class [looks just like the headers / footers here](https://cloudup.com/croSToB-tXq).

Without it, UITableView's default font (size + color) wouldn't match the design guidelines.